posts/2020-12-18-microprofile-long-running-actions-beta.adoc

```diff
@@ -77,7 +81,7 @@ In order to start a coordinator in Open Liberty you first need to enable the mpL
         <feature>mpLRACoordinator-1.0</feature>
     </featureManager>
    
-<!-- To access this server from a remote client add a host attribute to the following element, e.g. host="*" -->
+<!-- To access this server from a remote client, add a host attribute to the following element, for example, host="*" -->
```

Comment:
This text should not be changed as it is what is in the server.xml by default

Reply:
ok- sorry thought it was a comment on the example- reverted it